app/src/test/java/org/mozilla/focus/topsites/DefaultTopSitesStorageTest.kt

```diff
@@ -24,68 +25,71 @@ class DefaultTopSitesStorageTest {
     private val pinnedSitesStorage: PinnedSiteStorage = mock()
 
     @Test
-    fun `WHEN a top site is added THEN the pinned sites storage is called`() = runBlockingTest {
-        val defaultTopSitesStorage = DefaultTopSitesStorage(
-            pinnedSitesStorage,
-            coroutineContext
-        )
+    fun `WHEN a top site is added THEN the pinned sites storage is called`() =
+        runTest(UnconfinedTestDispatcher()) {
```

Comment:
Here and below: Can this code be moved to the above row, similar to how it was before?
Otherwise the git history gets unneededly polluted.